openmc/source.py

```diff
@@ -104,6 +104,14 @@ def strength(self, strength):
         self._strength = strength
 
     def to_xml(self):
+        """Return XML representation of the source
```

Comment:
I'm just trying my hardest to break everyone's code as my way of celebrating revolution and independence :-)